bitnami/jupyterhub/Chart.yaml

```diff
@@ -26,4 +26,4 @@ name: jupyterhub
 sources:
   - https://github.com/bitnami/bitnami-docker-jupyterhub
   - https://github.com/jupyterhub/jupyterhub
-version: 0.2.7
+version: 0.2.8
```

Comment:
Please bump a minor version (`0.3.0`). 
As this PR adds new features it is considered a minor and not a patch.